runtests.py

```diff
@@ -175,7 +176,7 @@ def add_stubs(driver: Driver) -> None:
 
     modules = {'typing'}
     # TODO: This should also test Python 2, and pass pyversion accordingly.
-    for version in ["2and3", "3", "3.3", "3.4", "3.5"]:
```

Comment:
I would have added "3.6" and "3.7", but that causes the tests to fail with the following:

```
FAILURE  #3 check stubs

mypy: can't find module 'contextvars'
```

This should also probably use a glob, so adding directories is not forgotten in the future.